Directory.Build.props

```diff
@@ -4,4 +4,13 @@
     <__XFBuildTasksLocation Condition="'$(__XFBuildTasksLocation)' == '' AND '$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory).nuspec\netstandard2.0\</__XFBuildTasksLocation>
     <__XFBuildTasksLocation Condition="'$(__XFBuildTasksLocation)' == ''">$(MSBuildThisFileDirectory).nuspec\net46\</__XFBuildTasksLocation>
   </PropertyGroup>
- </Project> 
+  <PropertyGroup>
+    <Company>Microsoft</Company>
+    <Product>Xamarin.Forms</Product>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(CI)' == ''">
```

Comment:
Generally a bummer when the `CI` build is different than the local build, no?

Reply:
The build is not different it just adds metadata if it's on CI or not , like the build number, branch or pr.